_gtfobins/systemctl.md

```diff
@@ -20,4 +20,9 @@ functions:
         WantedBy=multi-user.target' > $TF
         sudo systemctl link $TF
         sudo systemctl enable --now $TF
+    - description: |
+        Spawn interactive root shell escaping the pager.
```

Comment:
Use the following as a description (single line without `|`):

    This invokes the default pager, which is likely to be [`less`](/gtfobins/less/), other functions may apply.